app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
     <%= tag :meta, name: 'viewport', content: 'width=device-width, initial-scale=1' %>
     <%= tag :meta, property: 'og:image', content: asset_pack_path('media/images/govuk-opengraph-image.png') %>
     <%= tag :meta, name: 'theme-color', content: '#0b0c0c' %>
+    <%= tag :meta, charset: 'utf-8' %>
```

Comment:
This is required because percy is an idiot and doesn't take the charset from the headers when generating snapshots